<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RovDataSplines</class>
 <widget class="QDialog" name="RovDataSplines">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Data graphs</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="voltageTab">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string>Voltage graph</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QChartView" name="voltageChartView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="currentTab">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string>Current graph</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QChartView" name="currentChartView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="yawTab">
      <attribute name="title">
       <string>Yaw graph</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QChartView" name="yawChartView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="depthTab">
      <attribute name="title">
       <string>Depth graph</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QChartView" name="depthChartView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pitchTab">
      <attribute name="title">
       <string>Pitch graph</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QChartView" name="pitchChartView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="rollTab">
      <attribute name="title">
       <string>Roll graph</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QChartView" name="rollChartView"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
